python/semantic_kernel/kernel_extensions/inline_function_definitions.py

```diff
@@ -12,6 +11,7 @@
 from semantic_kernel.semantic_functions.semantic_function_config import (
     SemanticFunctionConfig,
 )
```

Comment:
Not for this PR, but this formatting is a hint that your module names are too long or your namespaces are too deeply nested. :-)

Reply:
Agreed, I think after we get through removing some more indirection and config classes, we could take a pass and clean up module names & nesting!